server/data_processors/DashboardDataProcessor/DashboardDataProcessor.test.js

```diff
@@ -0,0 +1,159 @@
+import DashboardDataProcessor from '.'
+import {
+  createDb,
+  createAnalysisConfig,
+  createSubjectDayData,
+  createAssessmentsFromConfig,
+  createMatrixData,
+} from '../../../test/fixtures'
+
+describe('Data Processors - Dashboard', () => {
+  describe(DashboardDataProcessor.calculateDashboardData, () => {
+    const assessmentsFromConfig = [
+      createAssessmentsFromConfig({
+        assessment: 'size_of',
+        collection:
+          '3da98187246398008970f684328c24421ea7c3a1246e82eb63cdf3134da85bb9',
```

Comment:
this is a nit, and fine to use these strings, but the collection can be anything like `size-of-collection` if we want to make the strings shorter or more obvious.

Reply:
ca15468